crates/astria-conductor/src/block_cache.rs

```diff
@@ -29,9 +29,9 @@ pub(crate) struct BlockCache<T> {
 impl<T> BlockCache<T> {
     /// Creates a new block cache.
     ///
-    /// The first block it will serve is at height 0.
+    /// The default first block is at height 1.
     pub(crate) fn new() -> Self {
-        Self::with_next_height(Height::from(0u32))
+        Self::with_next_height(Height::default())
```

Comment:
maybe just have an explicit height 1?

Reply:
Removed the `BlockCache::new` constructor and return an error if `BlockCache::with_next_height` is called with a height of zero.